include/openthread/platform/radio.h

```diff
@@ -276,10 +280,27 @@ typedef struct otRadioFrame
              * Note: `mTxDelayBaseTime` + `mTxDelay` SHALL point to the point in
              * time when the end of the SFD will be present at the local
              * antenna, relative to the local radio clock.
+             *
+             * If this field is non-zero, `mMaxCsmaBackoffs` should be ignored.
+             *
+             * This field does not affect CCA behavior which is controlled by `mCsmaCaEnabled`.
              */
             uint32_t mTxDelay;
 
-            uint8_t mMaxCsmaBackoffs; ///< Maximum number of backoffs attempts before declaring CCA failure.
+            /**
+             * Maximum number of CSMA backoff attempts before declaring channel access failure.
+             *
+             * This is applicable and MUST be used when radio platform provides the `OT_RADIO_CAPS_CSMA_BACKOFF`
+             * capability.
+             *
+             * This field MUST be ignored if `mCsmaCaEnabled` is set to `false` (i.e. CCA is disabled) or
```

Comment:
```suggestion
             * This field MUST be ignored if `mCsmaCaEnabled` is set to `false` (CCA is disabled) or
```

"i.e." not needed here